app/uk/gov/hmrc/apihubapplications/services/ApplicationsCredentialsService.scala

```diff
@@ -183,10 +186,32 @@ class ApplicationsCredentialsServiceImpl @Inject()(
     }
   }
 
+  override def fetchAllScopes(applicationId: String)(implicit hc: HeaderCarrier): Future[Either[ApplicationsException, Seq[CredentialScopes]]] = {
+    searchService.findById(applicationId, enrich = false).flatMap {
+      case Right(application) =>
+        Future.sequence(
+          Seq(Primary, Secondary).flatMap(
```

Comment:
Not that we care that much with the environment parity around the corner but
```suggestion
          EnvironmentName.values.flatMap(
```

Reply:
EnvironmentName should disappear with environment parity. That will be replaced by something dynamic driven by configuration.